src/integrationTest/java/uk/gov/hmcts/darts/transcriptions/controller/TranscriptionControllerAdminGetTranscriptionIntTest.java

```diff
@@ -333,6 +338,173 @@ void getTransactionsForUserBeyondOrEqualToDateNoUserSpecified() throws Exception
             .andReturn();
     }
 
+
+    @Test
+    void documentSearch_superUser_shouldSeeNotHiddenDocuments() throws Exception {
+        List<TranscriptionDocumentEntity> transcriptionDocumentResults = transcriptionDocumentStub.generateTranscriptionEntities(4, 1, false, true, false,
+                                                                                                                                 true);
+
+        superAdminUserStub.givenUserIsAuthorised(userIdentity);
+        when(userIdentity.userHasGlobalAccess(Set.of(SecurityRoleEnum.SUPER_USER))).thenReturn(true);
+        transcriptionDocumentResults.getFirst().setHidden(true);
+        transcriptionDocumentResults.getLast().setHidden(true);
+        dartsDatabase.save(transcriptionDocumentResults.getFirst());
+        dartsDatabase.save(transcriptionDocumentResults.getLast());
+
+        SearchTranscriptionDocumentRequest request = new SearchTranscriptionDocumentRequest();
+        request.setCaseNumber(transcriptionDocumentResults.getFirst().getTranscription().getHearing().getCourtCase().getCaseNumber());
+
+        MvcResult mvcResult = mockMvc.perform(post(ENDPOINT_DOCUMENT_SEARCH)
+                                                  .header("Content-Type", "application/json")
+                                                  .content(objectMapper.writeValueAsString(request)))
+            .andExpect(status().is2xxSuccessful())
+            .andReturn();
+
+        assertEquals(200, mvcResult.getResponse().getStatus());
+
+        JSONAssert.assertEquals(
```

Comment:
Should you not have a document in here that is hidden and make sure the super user can't see it?